website/core/DocsRating.js

```diff
@@ -5,11 +5,12 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import ExecutionEnvironment from '@docusaurus/ExecutionEnvironment';
 import React, {useState} from 'react';
+import useIsBrowser from '@docusaurus/useIsBrowser';
 
 const DocsRating = ({label}) => {
-  if (!ExecutionEnvironment.canUseDOM) {
+  const isBrowser = useIsBrowser();
+  if (!isBrowser) {
```

Comment:
This fixes a little React hydration issue. The issue was already there but is now logged thanks to React 18 new hydration callback `onRecoverableError(error)` (https://react.dev/blog/2022/03/29/react-v18)

The SSR/SSG render and the first CSR render outputs should match, so things such as `typeof window !== "undefined"` etc should be avoided.